README.md

```diff
@@ -21,7 +21,7 @@ Build and launch the node app with
 
 ```
 cd ./server
-npm install
+yarn
```

Comment:
@ao508 i changed this because we don't want people running multiple package manage managers in same project.  It will cause problems.  And Yarn is the way to go.